executor/distsql_test.go

```diff
@@ -269,3 +269,15 @@ func (s *testSuite3) TestIndexLookUpStats(c *C) {
 	stats.Merge(stats.Clone())
 	c.Assert(stats.String(), Equals, "index_task:4s, table_task:{num:4, concurrency:2, time:4s}")
 }
+
+func (s *testSuite3) TestIndexLookUpGetResultChunk(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists tbl")
+	tk.MustExec("create table tbl(a int, b int, c int, key(a))")
+	for i := 0; i < 101; i++ {
+		tk.MustExec(fmt.Sprintf("insert into tbl values(%d,%d,%d)", i, i, i))
+	}
+	tk.MustQuery("select * from tbl  where a >99 limit 1").Check(testkit.Rows("100 100 100"))
+	tk.MustQuery("select * from tbl  where a >10 limit 4,1").Check(testkit.Rows("15 15 15"))
```

Comment:
1. I have a test. If you don't analyze the table, the SQL will use TableScan. So that you can't test IndexScan.
2. I think the test is unstable because you don't know the data order. Use order by, so that you can make the result always stable.